pkg/catalog/routes.go

```diff
@@ -537,6 +534,61 @@ func (mc *MeshCatalog) buildInboundPolicies(t *access.TrafficTarget) []*trafficp
 	return inboundPolicies
 }
 
+func (mc *MeshCatalog) buildInboundAllowAllPolicies(svc service.MeshService) []*trafficpolicy.InboundTrafficPolicy {
+	inboundPolicies := []*trafficpolicy.InboundTrafficPolicy{}
+
+	hostnames, err := mc.getServiceHostnames(svc, true)
+	if err != nil {
+		log.Error().Err(err).Msgf("Error getting service hostnames for service %s", svc)
+		return inboundPolicies
+	}
+
+	servicePolicy := trafficpolicy.NewInboundTrafficPolicy(buildPolicyName(svc, false), hostnames)
+	weightedCluster := getDefaultWeightedClusterForService(svc)
+	svcAccounts := mc.kubeController.ListServiceAccounts()
+
+	// Build a rule for every service account in the mesh
+	for _, svcAccount := range svcAccounts {
+		sa := utils.SvcAccountToK8sSvcAccount(svcAccount)
+		servicePolicy.AddRule(*trafficpolicy.NewRouteWeightedCluster(wildCardRouteMatch, weightedCluster), sa)
+	}
+
+	if len(servicePolicy.Rules) > 0 {
+		inboundPolicies = append(inboundPolicies, servicePolicy)
+	}
+	return inboundPolicies
+}
+
+func (mc *MeshCatalog) buildOutboundAllowAllPolicies(srcServices []service.MeshService) []*trafficpolicy.OutboundTrafficPolicy {
+	outPolicies := []*trafficpolicy.OutboundTrafficPolicy{}
+
+	k8sServices := mc.kubeController.ListServices()
+	var destServices []service.MeshService
+	for _, k8sService := range k8sServices {
+		destServices = append(destServices, utils.K8sSvcToMeshSvc(k8sService))
+	}
+
+	// For allow all build an outbound policy to every service in the mesh execpt to itself
```

Comment:
Could you reword comments with `allow all` to `permissive mode` for consistency.